.github/workflows/ci.yml

```
@@ -259,9 +261,12 @@ jobs:
           yarn lerna:init
           yarn check:clean-workspace-after-install
 
-      - name: Build
+      - name: Install specific eslint version from matrix
         run: |
-          yarn build
+          yarn add -D -W eslint@${{ matrix.eslint-version}}
```

We'll want to `--ignore-scripts`, or else yarn will auto run the post-install which does a build. Just a minor optimisation to save ~10s on each CI run.

```suggestion
          yarn add -DW eslint@${{ matrix.eslint-version }} --ignore-scripts
```